### PR TITLE
Update Zcash mainnet network params due to Blossom upgrade

### PR DESCRIPTION
**Pull request by argjv**
[BLOCK-214](https://bitgoinc.atlassian.net/browse/BLOCK-214) New branch id id `0x2BB40E60`:
https://github.com/zcash/zips/blob/master/zip-0206.rst

### CODE REVIEW

**Comment by Xekyo** [comment]
> [BLOCK-214](https://bitgoinc.atlassian.net/browse/BLOCK-214) New branch id id `0x2BB40E60`:
> https://github.com/zcash/zips/blob/master/zip-0206.rst

Could you please add both those references to the commit message?

**Comment by Xekyo** [comment]
Given that you said the network id will only be valid after the hard fork has gone live, we should probably come up with some sort of strategy when to merge this. Also, I would recommend to bump the major version of this package instead of the minor in that case.

**Comment by argjv** [comment]
Platform will need a new PR (in progress), SDK and other dependencies will need to be rebuilt.
This is a mandatory change, not optional. Updating the major version will send the wrong message, updating the minor one will propagate the change more easily.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "bitgo-utxo-lib",
-  "version": "1.6.0",
+  "version": "1.6.1",
```
Comment (Xekyo) [inline package.json]: Still not a bugfix but a new feature, so should at least bump minor, not patch. ;)

Comment (Xekyo) [reply package.json]: Actually, I still think it should be the major, because downgrading from 1.6.1 to 1.6.0 would break Zcash.

Comment (argjv) [reply package.json]: Using 1.7.0

**Comment by argjv** [comment]
Squashed. We don't do major bumps even when we add new coins, so this does not merit one. The next major bump will be when we more to Typescript which will change the API.

**Comment by Xekyo** [comment]
Adding a new coin is adding a feature. Here, we're essentially ending support for the coin `Zcash (Sapling)` as well as adding support for a new coin `Zcash (Blossom)` which imho is a breaking change that predicates a major bump.

**Comment by Xekyo** [comment]
> The next major bump will be when we more to Typescript which will change the API.

Hasn't Bitcoinjs already ported most of the codebase to Typescript?

**Pull request merged**